index.js

```diff
@@ -184,7 +185,27 @@ function resolveWebpackPath(partial, filename, directory, webpackConfig) {
 
   try {
     var loadedConfig = require(webpackConfig);
-    var resolver = webpackResolve.create.sync(loadedConfig.resolve || {});
+    var resolveConfig = objectAssign({}, loadedConfig.resolve);
+
+    resolveConfig.modules = [];
+
+    if (resolveConfig.root) {
+      resolveConfig.modules = resolveConfig.modules.concat(resolveConfig.root);
+    }
+
+    if (resolveConfig.modulesDirectories) {
+      resolveConfig.modules = resolveConfig.modules.concat(resolveConfig.modulesDirectories);
+    }
+
+    var foundNodeModulesInPaths = resolveConfig.modules.some(function(dir) {
+      return dir.match('node_modules');
+    });
+
+    if (!foundNodeModulesInPaths) {
+      resolveConfig.modules.push('node_modules');
```

Comment:
Non-blocker: Looking at the webpack's docs. It seems to assume that a user should know that they're responsible for supplying `node_modules` in `modules` should they choose to supply custom values.

I'd only prefer to remove this config safety net for a few reasons:

1. https://github.com/dependents/node-filing-cabinet/pull/28/files#diff-168726dbe96b3ce427e7fedce31bb0bcR200 is in the hot code path. 
2. It would be a breaking change should we decide to remove this in the future due to perf. 
3. This also drops support for the `web_modules` default stated in https://webpack.github.io/docs/configuration.html#resolve-modulesdirectories. 
4. This doesn't allow a user to avoid node_modules if they wanted to.

Many of these are minor points though. I'm fine either way.